Core/Graphics/Source/DeviceImpl.cpp

```diff
@@ -25,7 +25,12 @@ namespace Babylon::Graphics
         m_state.Bgfx.Initialized = false;
 
         auto& init = m_state.Bgfx.InitState;
+#ifdef ANDROID
+        // on Android, having no window or context set the renderer API to no op. 
+        init.type = (config.Window || config.Device) ? s_bgfxRenderType : bgfx::RendererType::Noop;
+#else
         init.type = s_bgfxRenderType;
+#endif
```

Comment:
Ideally, this logic should go in the Android specific file instead of ifdef.

Reply:
Fixed